e2e/cypress/integration/messaging/message_parse_spec.js

```diff
@@ -18,7 +18,7 @@ describe('Messaging', () => {
         });
     });
 
-    it('M17444 - correctly parses "://///" as Markdown and does not break the channel', () => {
+    it('MM-T196 Markdown correctly parses "://///" and doesn\'t break the channel', () => {
```

Comment:
I haven't tried but maybe use backtick to avoid escaping `'`
```suggestion
    it(`MM-T196 Markdown correctly parses "://///" and doesn't break the channel`, () => {
```

Reply:
Yup, that will work but I'm good as is.

Reply:
Actually lint would fail on backtick, so nevermind 😄